skytap/resource_skytap_vm.go

```diff
@@ -217,12 +215,12 @@ func resourceSkytapVMCreate(d *schema.ResourceData, meta interface{}) error {
 	if err = addNetworkAdapters(d, meta, *vm.ID); err != nil {
 		return err
 	}
-	if err = addVMHardware(d, &client, &ctx, vm, environmentID, vmID); err != nil {
+	if err = addVMHardware(d, meta, &client, &ctx, vm, environmentID, vmID); err != nil {
 		return err
 	}
 
 	forceRunning := true
-	if os.Getenv("SKYTAP_FORCE_RUNNING") == "1" {
+	if os.Getenv("SKYTAP_DISABLE_FORCE_RUNNING") == "1" {
```

Comment:
as in previous we should not read any env variable from the provider directly without been declare in the config, an not changing the behaviour on testing

Reply:
As already noted: this is going to be addressed as part of #18